LICENSE.md

```diff
@@ -47,12 +54,11 @@ This license is separate from the license of any Mod it depends on and does not
 --------
 
 The User accepts the following liability:
- - The Mod is provided 'as is' with no warranties, implied or otherwise.
- - The Author is not responsible for dragon, troll, pirate, or ninja attacks.
+ - The Mod is provided as-is with no warranties, implied or otherwise.
  - Slaying any and all mythological creatures is the sole responsibility of the User.
- - The Author takes no responsibility for any damages incurred from the use or misuse of the Mod.
+ - The Author takes no responsibility for any damages incurred from the use or misuse of the Mod, direct or otherwise.
```

Comment:
What's the point of adding "direct or otherwise" here? It clarifies nothing, basically just saying "any damages of any kind" again.